src/main/scala/com/qubole/spark/hiveacid/reader/v2/HiveAcidInputPartitionV2.scala

```diff
@@ -0,0 +1,35 @@
+package com.qubole.spark.hiveacid.reader.v2
+
+import com.qubole.spark.hiveacid.rdd.HiveAcidPartition
+import com.qubole.spark.hiveacid.util.{SerializableConfiguration}
+import org.apache.spark.broadcast.Broadcast
+import org.apache.spark.sql.sources.v2.reader.InputPartition
+import org.apache.spark.sql.vectorized.ColumnarBatch
+import org.apache.hadoop.io.Writable
+import org.apache.hadoop.mapred.InputFormat
+import org.apache.hadoop.mapred.JobConf
+import org.apache.spark.sql.sources.v2.reader.InputPartitionReader
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.types._
+
+private[hiveacid] class HiveAcidInputPartitionV2(split: HiveAcidPartition,
```

Comment:
Instead of calling it HiveAcidInputPartitionV2, we should it ColumnarHiveAcidPartition.

Reply:
I have kept the name inline with "org.apache.spark.sql.sources.v2" .